doc/PythonSupport.md

```diff
@@ -0,0 +1,64 @@
+# Mystikos Support and Limitations for Python
+**NOTE**: this is a living document that is continually updated.
+
+## Verified/Supported python versions
+As stated in our [Getting started with Python](user-getting-started-docker-python.md) document, we recommend building your containerized python application using python 3.8.11 and 3.9.7. Note that if possible, try pin your python to a specific version like 3.8.11 instead of 3.8. Even when this document was written, there are newer python release than 3.8.11 or 3.9.7. We will do our best to stablize and support mystikos on newer versions.
```

Comment:
The get started doc mentions 3.8 and 3.9. Are you suggesting that 3.8.11 is more stable than 3.8?

Reply:
Unlike tag 3.8.11, tag 3.8 always "points to" the latest version. For example, when 3.8.12 comes out, 3.8 is updated to that version. Remember we had issues when the base image updates, and customer rebuilt the container, which caused failures.